src/asm/lexer.c

```diff
@@ -1031,10 +1038,11 @@ static void readLineContinuation(void)
 		} else if (c == '\r' || c == '\n') {
 			shiftChars(1);
 			if (c == '\r' && peek(0) == '\n')
+				/* Handle CRLF */
```

Comment:
For consistency, I'd rather have this before the `if`.

Reply:
Done.